### ISSUE TITLE
Getting 404 Error for controller MeetingCapture/GetUpcomingEvents

### ISSUE THREAD

**Issue by anupam-ujjwal** [microsoftgraph/meetings-capture-sample#17]
I have configured everything as per the guideline but for "MeetingCapture/GetUpcomingEvents" , I am getting 404 error. For other "POST" requests its coming correctly. 

https://mydomain/MeetingCapture/GetUpcomingEvents?teamId=db73a01e-b3f9-43a1-886b-84a28c99e614&channelId=19%3Aa0fc0a1a6ff84a33b951fabc6318c67a%40thread.tacv2

Note : mydomain is a dummy text. In actual we have created a separate domain.

**Comment by ravi16a87** [comment]
Same Here. I am also getting the same error. All the Get Methods are returning 404 error i.e. GetUpcomingEvents,GetRelevancePeople etc.

**Comment by nitbij** [comment]
> Same Here. I am also getting the same error. All the Get Methods are returning 404 error i.e. GetUpcomingEvents,GetRelevancePeople etc.

Hi @ravi16a87 , are you able to resolve this issue. I am also getting same issue.

**Comment by kandari7** [comment]
There might some runtime error in your code. if there is any error at the runtime the console shows 404 error. you can find the root cause by debugging the code.

**Comment by ikhwanmd** [comment]
Just a quick question, has anyone managed to get this up and running in their environment?

**Comment by akanimousoro** [comment]
I tried for months but couldn't get it up and running.Finally gave up

Sent from Yahoo Mail on Android 
 
  On Thu, 8 Oct 2020 at 7:54, ikhwanmd<notifications@github.com> wrote:   


Just a quick question, has anyone managed to get this up and running in their environment?

**Comment by ikhwanmd** [comment]
@akanimousoro, I would say that this app is awesome, but the instructions are not on point. But I managed to get it up and running although there are some bugs that needs to be rectified.

I've downloaded the latest code as of Oct 8th.
Deployed in my environment

Next I followed iiivanov1993 by fixing the graph api that i read at https://github.com/microsoftgraph/meetings-capture-sample/pull/7, but basically there were 3 files that need to be edited (GraphService.cs, CommunicationService.cs, Startup.cs) at https://github.com/microsoftgraph/meetings-capture-sample/pull/7/files
Remove the red lines and add/update the green ones
Deploy that again

In the database, i can see all tables being updated.
The meeting is created, i can see it in my calendar, in the PendingMeetings tab, Task in Planner

The bugs,
After creating the Meeting, it doesn't show it here

![image](https://user-images.githubusercontent.com/71329542/95430032-c5430d80-097d-11eb-8add-42ba6a8ccb80.png)

When it's time for the meeting, i don't have the Launch Meeting button like this

![image](https://user-images.githubusercontent.com/71329542/95430220-0affd600-097e-11eb-8eb7-1284f69692be.png)

and yeah, I didn't get any emails with regards to the meeting.
The Delete meeting also shows as being empty.

**Comment by isyedmud** [comment]
@ikhwanmd  I agree this is one of the best solutoin to have. Unfortunately i am also not able to make it work. Please can you share with me your working version of code link. I will be very grateful

Regards,
Syed
